week2/aniketsaoji/fizzbuzz.java

```diff
@@ -0,0 +1,28 @@
+import java.util.*;
+
+public class fizzbuzz {
+
+	public static int returnRandom() {
+		Random rand = new Random();
+		return rand.nextInt(100) + 1;	
+	}
+
+	public static void main (String [] args) {
+		int count = 1;
+		int randomNumber = returnRandom();
+		while (count < 101) {
+			while (randomNumber != count) {
+				randomNumber = returnRandom();
+			}
+			if ((randomNumber%3 == 0) && (randomNumber%5 == 0))
```

Comment:
You might want to include [horizontal whitespace](https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace) around `%` for readability:
`if ((randomNumber % 3 == 0) && (randomNumber % 5 == 0))`